lib/web/apiserver.go

```diff
@@ -2168,7 +2168,10 @@ func (h *Handler) logout(ctx context.Context, w http.ResponseWriter, sctx *Sessi
 	if err := sctx.Invalidate(ctx); err != nil {
 		return trace.Wrap(err)
 	}
-	websession.ClearCookie(w)
+	// Clear Web UI session cookie
+	websession.ClearCookie(w, websession.CookieName)
+	// Clear SAML IdP session cookie
+	websession.ClearCookie(w, websession.SAMLSessionCookieName)
```

Comment:
Do we always expect these to be cleared together? If yes, then maybe pull both clears into a single "clearSessionCookies" func and reuse it?

Reply:
https://github.com/gravitational/teleport/pull/37619/commits/96be22db00fce017e018a5f4f98176e2310c7627